adam_dunlap.cfdg

```diff
@@ -0,0 +1,34 @@
+startshape clock
+
+rule clock
+{
+    // A clock has an outer face, 2 hands, and a clock at each number
+    CIRCLE { a -0.5 }
+    hand { size 0.8 rotate 19 }
+    hand { rotate 225 }
+
+    offsetclock { rotate 0 }
```

Comment:
This seems like something that could have been done with either a loop or recursion, although to be honest I'm not seeing how right now.